### PR TITLE
11 create the map view

### LINKED ISSUE #11: Create the Map view
- [x] Create the Map view that we go to when selecting the map icons in the home page and in the artwork page
- [x] Display the closest artworks in the map
- [x] Display the user location in the map
If getting device location is not possible, do a fixed location for now (same as homepage (#9) )
- [x] Group artworks that have the same location with a special icon and location name if defined. 
- [x] Must follow mockups
- [x] No filters at the moment (later issue)

### PR DESCRIPTION
**Pull request by Tomansion**
## Describe your changes
Added map 
Placed user on map
Display artworks that are on screen

## Screenshots (if appropriate):
image
image

## Issue ticket number
Closes #11 and #26 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have written unit tests for my code if applicable

### CODE REVIEW

File: back/services/arangoManager.py
Comment (Smartycrouton) [inline back/services/arangoManager.py:1]: Ligne 136 : print(cursor) is usefull or it's just for testing ?

File: front/package.json
Comment (Smartycrouton) [inline front/package.json:1]: Why have you changed the version ?

File: front/package-lock.json
Comment (Smartycrouton) [inline front/package-lock.json:1]: Isn't it haram to push package-lock.json ?

File: back/tests/test_arango_manager.py
Comment (Smartycrouton) [inline back/tests/test_arango_manager.py:1]: Why have you take-off the descriptions ?

**Review by travisstephane — APPROVED** [review]

File: front/package-lock.json
Comment (Tomansion) [reply front/package-lock.json:1]: Nop, it's good to have it up to date for everyone

File: back/tests/test_arango_manager.py
Comment (Tomansion) [reply back/tests/test_arango_manager.py:1]: Not useful for the tests, triggers typo because french

File: front/package.json
Comment (Tomansion) [reply front/package.json:1]: What version ? The general APP version ?
I did not...

File: back/api.yaml [hunk 1]
```diff
@@ -53,3 +100,14 @@ components:
       required:
         - name
         - position
+
+    WorkWithDistance:
```
Comment (Tomansion) [inline back/api.yaml]: To remove

File: back/services/arangoManager.py
Comment (Tomansion) [reply back/services/arangoManager.py:1]: Well seen, will fix soon

**Comment by Tomansion** [comment]
To add : Smooth transition on markers open & close

File: back/tests/test_arango_manager.py
Comment (Smartycrouton) [reply back/tests/test_arango_manager.py:1]: Always the french....

Comment (Tomansion) [reply back/tests/test_arango_manager.py:1]: image

**Review by altristid — APPROVED** [review]
Greay job, we might have to work together on the arango part for a  next merge request !

**Review by Smartycrouton — APPROVED** [review]
Nothing to declare, good job

**Pull request merged**